uaclient/mainwindow.py

```diff
@@ -139,44 +145,15 @@ def _update_event_model(self, event):
         self.model.appendRow([QStandardItem(str(event))])
 
 
-class DataChangeUI(object):
+class DataChangeSubscriptionManager(object):
```

Comment:
No need to inherit from `object` in Python 3:

```suggestion
class DataChangeSubscriptionManager:
```